src/pudl/transform/ferc1.py

```diff
@@ -1520,7 +1549,9 @@ def add_corrections(
         f"{num_records} original records."
     )
 
-    return pd.concat([calculated_df, corrections], axis="index")
+    return pd.concat(
+        [calculated_df.convert_dtypes(), corrections.convert_dtypes()], axis="index"
```

Comment:
Is there a reason to do dtypes conversions here and not further upstream?

Reply:
this has to do with #3125 but i actually didn't even do this right